integration/tests/cook/test_basic.py

```diff
@@ -1413,7 +1373,6 @@ def test_change_failed_retries(self):
         finally:
             util.kill_jobs(self.cook_url, jobs)
 
-    @unittest.skipIf(util.has_one_agent(), 'Test requires multiple agents')
```

Comment:
This check is not needed -- it's acceptable to assume that tests must be run in environments with more than one node